src/components/LandingPage/LandingPage.scss

```diff
@@ -11,52 +11,134 @@
   @include flex (column, center, center);
 }
 
-.winner-title,
-.winner-name,
-.welcome-message,
-.welcome-message-header,
-.submission-dates-header,
-.submission-dates-april,
-.submission-dates-may,
-.hero-button {
-  font-size: 2.5vh;
+// .winner-title,
+// .winner-name,
+// .hero-button 
+.greeting-message,
+.welcome-message-header
+{
+  font-size: 1.3rem;
   width: 80%;
   text-align: center;
-  // position: absolute;
-  // top: 30%;
-  // left: 50%;
-  // transform: translate(-50%, -50%);
   color: white;
+  margin-bottom: 10px;
+  margin-top: 10px;
+  font-weight: bolder;
 }
 
 .winner-container {
-  @include flex(column, center, center);
-  width: 75%;
-  margin: 50px;
+  // @include flex(column, center, center);
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    flex-direction: column;
+    justify-content: space-around;
```

Comment:
Just noticed two `justify-content`'s here and curious about the commented out `mixin` above.